openerp/addons/base/res/res_company.py

```diff
@@ -311,24 +91,242 @@ def _get_header(self,cr,uid,ids):
+    rml_header2 = fields.Text(string='RML Internal Header', required=True, default=_header2)
+    rml_header3 = fields.Text(string='RML Internal Header for Landscape Reports', required=True, default=_header3)
+    rml_footer = fields.Text(string='Report Footer', help="Footer text displayed at the bottom of all reports.")
+    rml_footer_readonly = fields.Text(related='rml_footer', string='Report Footer', readonly=True)
+    custom_footer = fields.Boolean(help="Check this to define the report footer manually. Otherwise it will be filled in automatically.")
+    font = fields.Many2one('res.font', string="Font", default=lambda self: self._get_font(),
+                           domain=[('mode', 'in', ('Normal', 'Regular', 'all', 'Book'))],
+                           help="Set the font into the report header, it will be used as default font in the RML reports of the user company")
+    logo = fields.Binary(related='partner_id.image', default=_get_logo)
+    # logo_web: do not store in attachments, since the image is retrieved in SQL for
+    # performance reasons (see addons/web/controllers/main.py, Binary.company_logo)
+    logo_web = fields.Binary(compute='_compute_logo_web', store=True)
+    currency_id = fields.Many2one('res.currency', string='Currency', required=True, default=lambda self: self._get_euro())
+    user_ids = fields.Many2many('res.users', 'res_company_users_rel', 'cid', 'user_id', string='Accepted Users')
+    account_no = fields.Char(string='Account No.')
+    street = fields.Char(compute='_compute_address', inverse='_inverse_street')
+    street2 = fields.Char(compute='_compute_address', inverse='_inverse_street2')
+    zip = fields.Char(compute='_compute_address', inverse='_inverse_zip')
+    city = fields.Char(compute='_compute_address', inverse='_inverse_city')
+    state_id = fields.Many2one('res.country.state', compute='_compute_address', inverse='_inverse_state', string="Fed. State")
+    bank_ids = fields.One2many('res.partner.bank', 'company_id', string='Bank Accounts', help='Bank accounts related to this company')
+    country_id = fields.Many2one('res.country', compute='_compute_address', inverse='_inverse_country', string="Country")
+    email = fields.Char(related='partner_id.email', store=True)
+    phone = fields.Char(related='partner_id.phone', store=True)
+    fax = fields.Char(compute='_compute_address', inverse='_inverse_fax')
+    website = fields.Char(related='partner_id.website')
+    vat = fields.Char(related='partner_id.vat', string="Tax ID")
+    company_registry = fields.Char()
+    rml_paper_format = fields.Selection([('a4', 'A4'), ('us_letter', 'US Letter')], string="Paper Format", required=True, default='a4', oldname='paper_format')
+
+    _sql_constraints = [
+        ('name_uniq', 'unique (name)', 'The company name must be unique !')
     ]
+
+    @api.multi
+    def _compute_address(self):
```

Comment:
@rgo-odoo in response to your question: omitting the `@api.depends` decorator is not good practice. Non-stored computed fields are kept in cache after they have been computed. Without the decorator, the cache will never be invalidated in the current request, even if what the field depends on has been modified.